tests/main/snap-confine/task.yaml

```diff
@@ -39,3 +39,12 @@ execute: |
     echo "But nvidia files are gone"
     ! test -f /run/udev/tags/snap_test-snapd-tools_echo/+module:nvidia
     ! test -f /run/udev/tags/snap_test-snapd-tools_echo/+module:nvidia_modeset
+
+    echo "Ensure apparmor profile for snap-confine is parsable"
```

Comment:
You'll need to skip this part for fedora as there's no AppArmor. Looking at the tests, it's likely that debian-9 image is not using a kernel that supports AppArmor

Reply:
You can always parse the profile and just not load it into the kernel. `apparmor_parser --skip-kernel-load`

Reply:
On debian also failing. 
ERROR: Cache read/write disabled: interface file missing. (Kernel needs AppArmor 2.4 compatibility patch.)

Reply:
In fedora it is failing because the command apparmor_parser does not exist